src/app/pages/ConnectorsPage/components/ConnectorInstancesTable/ConnectorInstancesTable.tsx

```diff
@@ -137,7 +138,7 @@ export const ConnectorInstancesTable: FunctionComponent<
               onSearch: onSearchName,
               onRemoveChip: onRemoveNameChip,
               onRemoveGroup: onRemoveNameChipGroup,
-              validate: (_val) => true,
+              validate: (value) => validateSearchField(value),
```

Comment:
You can even do `validate: validateSearchField` I think if you want, no lambda needed.